src/components/ContactsList/Contacts/ContactName.jsx

```diff
@@ -1,21 +1,28 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-const ContactName = ({ firstname, lastname }) => (
-  <div className="u-ellipsis">
-    <span className="contact-firstname">{firstname}</span>
-    &nbsp;
-    <span className="contact-lastname">{lastname}</span>
-  </div>
-)
+const ContactName = ({ displayName, lastname }) => {
+  const namesToDisplay = (displayName && displayName.split(' ')) || []
+  return (
+    <div className="u-ellipsis">
+      {namesToDisplay.map((name, key) => (
+        <span key={`display-${key}`}>
+          <span className={name === lastname ? 'contact-lastname' : 'u-ml-1'}>
+            {name}
+          </span>
+          &nbsp;
```

Comment:
why this? don't the `u-ml-1` is there to handle the space between names?

Reply:
the space between names should be unbreakable. Besides the margin is only used to make space between avatar and name.

Reply:
>Besides the margin is only used to make space between avatar and name. 

OK but if the lastname is the first name to be displayed, we'll not have this margin. 

The margin should not be attached to this node, but its parent ?

Reply:
Yes you're right, fixed !